### DISCUSSION TITLE
55E90DE7-9BB3-4B0B-B407-975C26721259

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#162] [General]
# 55E90DE7-9BB3-4B0B-B407-975C26721259

Kepler-411 (KOI-1781) is a binary star system consisting of a K-type star and an M-type star, located 503.459 light-years away from Earth. It is known that four planets orbit the main star A (one super Earth and three sub Neptune objects)

https://celestia.mobi/addon?item=55E90DE7-9BB3-4B0B-B407-975C26721259

**Comment by exocritic5457** [comment]
I get that planets are exposed to a lot of radiation from their host stars, but I didn't think they meant to the extent that they're glowing neon green because of it.

**Reply by Galaxy-Leaper1** [reply]
This is indeed a problem. When making these textures, I did not refer to their temperature. Don't worry, I will update you later
